src/api/firebase.ts

```diff
@@ -95,6 +95,7 @@ export function useShoppingLists(user: User | null) {
 const ListItemModel = t.type({
 	id: t.string,
 	name: t.string,
+	itemQuantity: t.number,
```

Comment:
praise: nice use of expanding the exiting model!

Reply:
Yes! io-ts looks very unfriendly at first put with a bit of practice it becomes less-so! I highly recommend anyone here take a look into the documentation. Feel free to shoot me any questions about it 😃  getting comfortable with it will help a ton

Reply:
Thank you for noticing! For a few minutes there, I was all, "Is this the data model? This has got to be the data model schema thing!"